encode/av1encode.c

```diff
@@ -806,7 +806,8 @@ static void process_cmdline(int argc, char *argv[])
         else {
             struct stat tmp;
 
-            fstat(fileno(srcyuv_fp), &tmp);
+            VAStatus va_status = fstat(fileno(srcyuv_fp), &tmp);
```

Comment:
Return value of `fstat` is unrelated to `VAStatus`. You can't directly assign. And since you have a lot of such cases below - introduce new macro:
```
#define CHECK_FN(cod, func)                      \
    if(!(cod))                                                      \
    {                                                           \
        fprintf(stderr,"%s:%s (%d) failed, exit\n", __func__, func, __LINE__); \
        exit(1);                                                \
    }

<...>

int ret = fstat(fileno(srcyuv_fp), &tmp);
CHECK_FN(ret, "fstat");
```

Reply:
Already used new macro and modified cases of same type, thanks!